back/engines/commercial/posthog_integration/app/services/posthog_integration/track_posthog_service.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+class PosthogIntegration::TrackPosthogService
+  def initialize(posthog_client = POSTHOG_CLIENT)
+    @posthog = posthog_client
+  end
+
+  def identify_user(user)
+    return unless @posthog && track_user?(user)
+
+    # The information passed to identify best stays in sync with the same call
+    # happening in the FE, currently happening in
+    # front/app/modules/commercial/posthog/index.ts:49
```

Comment:
That's a bit annoying. Can't the front-end just use a subset of those attributes (identifiers)?